frontends/mit-open/src/pages/PrivacyPage/PrivacyPage.tsx

```diff
@@ -248,6 +198,16 @@ const PrivacyPage: React.FC = () => {
             maintains, or transmits. Nonetheless, no method of transmission over
             the Internet or method of electronic storage is 100% secure, and
             therefore we do not guarantee its absolute security.
+            <br />
+            <br />
```

Comment:
```suggestion
            </BodyText>
            <BodyText variant="body1">
```

The general recommendation I see is to use `p` for paragraphs and to consider `<br>` only things like lyrics, poetry, and addresses. (We use `<br>` for addresses elsewhere on this page, incidentally.)